sensor-system/migrations/v0.2.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE Location (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(32) NOT NULL,
+    address VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE Device (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(32) NOT NULL,
+    location_id INTEGER NOT NULL,
+    status VARCHAR(16) DEFAULT 'active',
+    FOREIGN KEY (location_id) REFERENCES Location(id) ON DELETE CASCADE
```

Comment:
```suggestion
CREATE TABLE Device (
    "id" SERIAL PRIMARY KEY,
    "name" VARCHAR(32) NOT NULL,
    "locationId" INTEGER NOT NULL,
    "status" VARCHAR(16) DEFAULT 'active',
    "created" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
    "updated" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
    "deleted" TIMESTAMP WITH TIME ZONE,
    FOREIGN KEY (locationId) REFERENCES Location(id) ON DELETE CASCADE
```

Again wrap every column definition in quotes.

Use camel case for column names, while not being convention it's easier to handle once we query the data because we can access the property like `row.locationId` instead of `row["location_id"]` Typically when columns have underscored names after the data is queried, it gets remapped to a camel cased name at the data access layer.

What is status here? active or inactive? You could just make it a boolean value.